pyaedt/modeler/Modeler.py

```diff
@@ -412,32 +430,26 @@ def create(self, origin=None, reference_cs='Global', name=None,
 
     @property
     def orientation(self):
-        """
-        Construct the internal Named Array for orientation
-        """
+        """Coordinate system orientation."""
```

Comment:
```suggestion
        """Internal named array for orientation."""
```
This will be made to be private at some point.